app/user.go

```diff
@@ -2236,15 +2236,32 @@ func (a *App) PromoteGuestToUser(user *model.User, requestorId string) *model.Ap
 
 	promotedUser, err := a.GetUser(user.Id)
 	if err != nil {
-		return err
+		mlog.Error(err.Error())
 	}
 
-	message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_USER_UPDATED, "", "", "", nil)
-	message.Add("user", promotedUser)
-	a.Publish(message)
-
+	a.sendUpdatedUserEvent(*promotedUser)
```

Comment:
I assume invoking this with a `nil` `*promotedUser` is OK?

Reply:
actually not, would be super weird because it should fail before reach this point, but is a case that can happen, so I prefer to fix it and avoid any possibility of a nil pointer access.

Reply:
Done